src/hooks/useCheckProjectPriceStatus .ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@tanstack/react-query';
+import { getTokenPriceRangeStatus } from '@/services/tokenPrice.service';
+
+export const useCheckProjectPriceStatus = (
+  allProjects: any,
+  allRounds: any,
+) => {
+  return useQuery({
+    enabled: !!allProjects && !!allRounds,
+    queryKey: ['projectPriceStatus', allProjects, allRounds],
```

Comment:
`allProjects` and `allRounds` types are not of string type, right? If so, having them in the cache key is inefficient! It will cause the query to re-execute again and again.
Maybe the `projectPriceStatus` is enough since all the projects and all the rounds would not change.